src/ddspdrum/module.py

```diff
@@ -365,12 +366,27 @@ def __init__(
         self.amp_envelope = amp_adsr(sustain_duration)
         self.vco = vco
         self.vca = vca
+        self.noise_ratio = noise_ratio
```

Comment:
Should we call this vco_noise_ratio?

Reply:
It's not a vco though, it's part of the drum module.

Reply:
Oh I just meant it's the noise ratio applied to the vco bank. Okay